plugins/emmc/fu-emmc-device.c

```diff
@@ -214,6 +223,7 @@ fu_emmc_device_probe(FuDevice *device, GError **error)
 	fu_device_add_instance_u16(device, "OEM", oemid);
 	fu_device_build_instance_id(device, NULL, "EMMC", "MAN", "OEM", NULL);
 	fu_device_build_instance_id(device, NULL, "EMMC", "MAN", "OEM", "NAME", NULL);
+	fu_device_build_instance_id(device, NULL, "EMMC", "MAN", "NAME", "REV", NULL);
```

Comment:
can we do `"EMMC", "MAN", "OEM", "NAME", "REV"` instead? In that way it's a complete superset of the line above.

Reply:
Added as an additional. In fact, I am a bit unhappy about OEMID being a part of GUID in the first place - cuz that is not something that identifies the storage device per-ce, but not gonna remove it not to break potential dependency on it by those who added the parameter.

Reply:
It's quite common to have custom firmware per OEM though. But usually this corresponds with a custom part number too..

Reply:
True. Probably it is a good idea to have both then - in the case which I am concern with OEM ID should be ignored and only model + rev needs to be matched.
There is another complication (not specific to OEMID though) - if have same FW number for all part in a family, but different model tags for different sizes, currently I would need to mention each mapping for each model, is it right? should I thin add may be just REV tag as well?